ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala

```diff
@@ -333,15 +355,13 @@ object PerformanceEnvelope {
 
   /** Throughput test
     *
-    * @param numPings  how many pings to run during the throughput test
-    * @param maxInflight how many inflight commands we can have. set it high enough such that the system saturates, keep it low enough to not hit timeouts.
+    * @param numPings       how many pings to run during the throughput test
     * @param numWarmupPings how many pings to run before the perf test to warm up the system
     */
   class ThroughputTest(
       val logger: Logger,
       val envelope: Envelope,
       val numPings: Int = 200,
-      val maxInflight: Int = 40,
```

Comment:
Was this unused?

Reply:
@dasormeter : also as we are at it: we probably should compute the number of pings as well from the Envolope. There the goal I'd see is that we get a stable measurement of the throughput. Which means we'd like to observe at least some `X` number of requests sent while maximum throughput is reached.

To see why a constant does not suffice, think about a throughput and latency Envolope that requires more than 200 requests to be in flight at the same time; e.g., the Enterprise one.

So, a better value is to set  `numPings = 10 * maxRequestsInFlight`, which means we send enough pings to observe at least 10 flights of requests. For statistical significance 10 is probably rather on the low side. It matches though quite closely the 200 pings assumption that @daravep took in his original code.

If we see a lot of variation in our measurements, then we should upgrade the 10.

Please also add a comment in the code capturing the reasoning in this comment, if/when you change the code.